recipes/concurrentloghandler/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "ConcurrentLogHandler" %}
+{% set version = "0.9.1" %}
+{% set sha256sum = "8225a590fd4194c413fa26675bde5f6b80ad79e4182d5876ba3e264f77755918" %}
+
+package:
+  name: {{ name.lower() }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256sum }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - pywin32  # [win]
+
+test:
+  imports:
+    - cloghandler
+
+about:
+  home: https://launchpad.net/python-concurrent-log-handler
+  license: Apache 2
+  # license_file: LICENSE.rst
```

Comment:
Please manually include the license file. See:

http://conda-forge.github.io/docs/meta.html#packaging-the-license-manually